packages/wrangler/src/__tests__/helpers/mock-console.ts

```diff
@@ -95,3 +95,12 @@ export function stripTrailingWhitespace(str: string): string {
 function replaceByte(stdout: string): string {
 	return stdout.replaceAll(/.[0-9][0-9] KiB/g, "xx KiB");
 }
+
+/**
+ * Replacing version number with a placeholder of form x.x.x
+ *
+ * Use this in snapshot tests to be resilient to changes in version.
+ */
+export function replaceVersion(stdout: string): string {
+	return stdout.replaceAll(/\d+\.\d+\.\d+/g, "x.x.x");
```

Comment:
idea: maybe you could read the version from package.json and match the version directly here instead of a regex pattern?

Reply:
you could potentially borrow/reference some logic from my `version-script.js` file in the `.github` directory.

Reply:
Fixed in https://github.com/cloudflare/wrangler2/pull/1498/commits/c3457ea6d0c4c4df74fca7bfdee0333525eb9e77

I didn't see any advantage to using the logic from `.github/version-script.js` when we can just import the version directly with es6 import

Reply:
delete this function now that we don't need/use it?